app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -68,6 +68,7 @@ class MainActivity : AppCompatActivity() {
         navigation.setOnNavigationItemSelectedListener(listener)
 
         supportActionBar?.title = "Events"
+        loadFragment(supportFragmentManager, EventsFragment(), frameContainer.id)
```

Comment:
This is unconditionally loading it.

Reply:
Can I move it in onCreate(), because it is called only once when the app starts?

Reply:
There are if conditions below, add it there

Reply:
Updated